src/go/rpk/pkg/cli/cmd/cloud/login.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package cloud
+
+import (
+	"errors"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/vcloud"
+)
+
+var (
+	ErrAuthorizationFailed = errors.New("authorization failed, please retry")
+)
+
+func NewLoginCommand(fs afero.Fs) *cobra.Command {
+	return &cobra.Command{
+		Use:   "login",
+		Short: "Login to vectorized cloud",
+		Long:  `Authorize rpk to access vectorized cloud with your user credentials.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			auth0Client := vcloud.NewDefaultAuth0Client()
+			_, err := vcloud.Login(auth0Client)
+			if err != nil {
+				log.Debug(err)
+				return ErrAuthorizationFailed
```

Comment:
This could also be an HTTP error, not necessarily always auth, right?

Reply:
yeah, but to the user, do that make a difference? My thought was that they could just retry - if they are tech savy, they could use `-v` and get more info that way. WDYT?

Reply:
I do think it matters, and I think it's useful to know why things fail without enabling verbose output.

Reply:
done, I print the error at all times, there's really no "not authorized" here, user either finishes the auth or does not so there's nothing more meaningful I can put there